src/hw-2/index.ts

```diff
@@ -0,0 +1,15 @@
+import express from 'express';
+import * as core from 'express-serve-static-core';
```

Comment:
I don't think you really need this package and this type

Reply:
Deleted.

Reply:
I mean you have typescript interfaces in main package 'express', you don't need this on with `serve-static` until you decide to return static files of some folder.